android/assets/jsons/translationsByLanguage/Traditional_Chinese.properties

```diff
@@ -2,33 +2,35 @@
 # Tutorial tasks
 
  # Requires translation!
-Move a unit!\nClick on a unit > Click on a destination > Click the arrow popup = 
+Move a unit!\nClick on a unit > Click on a destination > Click the arrow popup = 移動你的人民!\點擊一個單位 > 點擊其目的地 > 點擊確認箭頭前進
  # Requires translation!
-Found a city!\nSelect the Settler (flag unit) > Click on 'Found city' (bottom-left corner) = 
+Found a city!\nSelect the Settler (flag unit) > Click on 'Found city' (bottom-left corner) = 建立你的首都!\n選取移民(旗子圖樣單位) > 點擊"建立首都"(螢幕左下角)
  # Requires translation!
-Enter the city screen!\nClick the city button twice = 
+Enter the city screen!\nClick the city button twice = 進入城市介面!\n點擊城市兩下
  # Requires translation!
 Pick a technology to research!\nClick on the tech button (greenish, top left) > \n select technology > click 'Research' (bottom right) = 
+選一個科技開始研究!\n點擊科技按鈕(左上方,綠色按鈕) > \n選擇一個科技 > 點擊"開始研究" (右下角) 
  # Requires translation!
-Pick a construction!\nEnter city screen > Click on a unit or building (left side) = 
+Pick a construction!\nEnter city screen > Click on a unit or building (left side) = 選一個設施開始建造!\n進入城市介面 > 點擊一個單位或建築 (畫面左邊)
  # Requires translation!
-Pass a turn!\nCycle through units with 'Next unit' > Click 'Next turn' = 
+Pass a turn!\nCycle through units with 'Next unit' > Click 'Next turn' = 跳過一回合!\n用"下一個單位"來循環你的單位 > 點擊"下一回合"
  # Requires translation!
 Reassign worked tiles!\nEnter city screen > click the assigned (green) tile to unassign > \n click an unassigned tile to assign population = 
+重新分配勞動人口!\n進入城市介面 > 點擊已被分配勞力的領域已取消分配 > \n 接著點擊未被分配勞力的領域已分配勞動人口
```

Comment:
```suggestion
```
(moved to line above, since translation needs to be in the same line as the original.)